### ISSUE TITLE
Deprecated `from numpy.testing.decorators import skipif`

### ISSUE THREAD

**Issue by SebastianoF** [nipy/nibabel#718]
Hello!

Thanks for nibabel, I have been using it extensively in the last few years!
Today I updated to numpy 1.15.0 and the following warning have started to appear when running tests on python 3:

```
========================================================= warnings summary ==========================================================
/Applications/python_virtualenvs/clean3/lib/python3.6/site-packages/nibabel/testing/__init__.py:21
  /Applications/python_virtualenvs/clean3/lib/python3.6/site-packages/nibabel/testing/__init__.py:21: DeprecationWarning: Importing from numpy.testing.decorators is deprecated since numpy 1.15.0, import from numpy.testing instead.
    from numpy.testing.decorators import skipif

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```
To solve the issue, I believe the line nibabel/testing/__init__.py:21
```
from numpy.testing.decorators import skipif
```
should be substituted with something like:
```
from numpy.testing import dec
skipif = dec.skipif
```

Thanks!

**Comment by effigies** [comment]
Fixed in #705 (merged just now - thanks for the reminder!)

**Comment by SebastianoF** [comment]
👍 Thanks!
